src/main/java/de/progen_bot/commands/user/Say.java

```diff
@@ -1,32 +1,43 @@
 package de.progen_bot.commands.user;
 
+import java.time.Instant;
+
 import de.progen_bot.command.CommandHandler;
 import de.progen_bot.command.CommandManager.ParsedCommandString;
 import de.progen_bot.db.entities.config.GuildConfiguration;
+import de.progen_bot.listeners.SayListener;
+import de.progen_bot.listeners.SayModel;
 import de.progen_bot.permissions.AccessLevel;
+import net.dv8tion.jda.api.EmbedBuilder;
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
 
-
 public class Say extends CommandHandler {
 
     public Say() {
         super("say", "say <text>", "let the bot write a message");
     }
 
     @Override
-    public void execute(ParsedCommandString parsedCommand, MessageReceivedEvent event, GuildConfiguration configuration) {
-        event.getMessage().delete();
+    public void execute(ParsedCommandString parsedCommand, MessageReceivedEvent event,
+            GuildConfiguration configuration) {
+
         StringBuilder out = new StringBuilder(" ");
         for (String s : parsedCommand.getArgs()) {
             out.append(s).append(" ");
         }
```

Comment:
Can also be replaced by ```java
final var out String.join(" ", parsedCommand.getArgs())
```

Reply:
Nice solution! This should be used in some other classes too.